server/app/services/migration/ProgramMigrationService.java

```diff
@@ -13,22 +13,33 @@
 import com.google.common.collect.ImmutableSet;
 import com.google.inject.Inject;
 import controllers.admin.ProgramMigrationWrapper;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import models.ProgramNotificationPreference;
 import repository.QuestionRepository;
 import services.ErrorAnd;
+import services.LocalizedStrings;
 import services.program.ProgramDefinition;
 import services.program.ProgramType;
 import services.question.exceptions.UnsupportedQuestionTypeException;
 import services.question.types.QuestionDefinition;
 import services.question.types.QuestionDefinitionBuilder;
+import services.question.types.QuestionDefinitionConfig;
+import services.question.types.TextQuestionDefinition;
 
 /**
  * A service responsible for helping admins migrate program definitions between different
  * environments.
  */
 public final class ProgramMigrationService {
+  private static final String CONFLICTING_QUESTION_FORMAT =
+      "%s - %s"; // this will transform to a key formatted like `%s__%s`
+  private static final Pattern SUFFIX_PATTERN = Pattern.compile(" - [a-z]+$");
+  private static final String ALPHABET = "abcdefghijklmnopqrstuvwzyz";
```

Comment:
```suggestion
  private static final String ALPHABET = "abcdefghijklmnopqrstuvwxyz";
```

Missing `x` and have `z` twice